CODE_OF_CONDUCT.md

```diff
@@ -1,46 +1,9 @@
-# Contributor Covenant Code of Conduct
+Welcome to the Weblate community
 
-## Our Pledge
+Accept and offer criticism constructively.
+Let anyone have the privacy they desire.
 
-In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
+Settle differences within these boundaries.
 
-## Our Standards
-
-Examples of behavior that contributes to creating a positive environment include:
-
-* Using welcoming and inclusive language
-* Being respectful of differing viewpoints and experiences
-* Gracefully accepting constructive criticism
-* Focusing on what is best for the community
-* Showing empathy towards other community members
-
-Examples of unacceptable behavior by participants include:
-
-* The use of sexualized language or imagery and unwelcome sexual attention or advances
-* Trolling, insulting/derogatory comments, and personal or political attacks
-* Public or private harassment
-* Publishing others' private information, such as a physical or electronic address, without explicit permission
-* Other conduct which could reasonably be considered inappropriate in a professional setting
```

Comment:
Professionalism for the sake of professionalism is bad. The professional world is full of concessions away from humanity, which doesn't befit people, but rather corporate identity and decision-making. I will never bow to this. My power is mine, and I use it to question the age in which this was acceptable. If then not to all, it is clear to me that a historically changing landscape, which in no way has found stability at present day is a model to admire. Work is called work because it is tedious, this is what makes it so.